content/swiftui/concepts/viewmodifier/terms/underline/underline.md

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
There are a lot of extra line breaks that need to be removed now.

- The line breaks between lines in the metadata need to be removed.
- In the body of the article there should not be more than one line break in a row.
- Extra line breaks in the code blocks need to be removed.
- There should be one line break (only one) before and after paragraphs, headers and code blocks.

Reply:
All extra line breaks were deleted :)